tests/unit/Http/ResponseDecoderTest.php

```diff
@@ -96,4 +97,34 @@ public function shouldThrowExceptionWhenJsonWithInvalidDataIsDecoded(): void
         $this->expectExceptionMessage('"invalid": [],');
         $this->decoder->decode($response);
     }
+
+    /** @test */
+    public function shouldDecodeVersionOneRecommendationResponse(): void
```

Comment:
So is there backward compatibility or not? I thought once you use library with this updated code, you will get response in the new format (that is why it is BC break, right?). If you could still get array of string from `getData`, how could you safely update you app for the new format?

Reply:
You have a point here; there is a backwards compatibility but we don't want to expose it in the PHP client, as it will get removed from Matej at some point. I've updated the Client, so that it's a clean BC-break.

Reply:
So will you drop this testcase and also support for the old format :)? (and also response-recommendation-v1.json fixture )